docs/tutorial-01.md

```diff
@@ -0,0 +1,210 @@
+    This is a new container from ipbabble [ 7 ]
+    This is a new container from ipbabble [ 8 ]
+    This is a new container from ipbabble [ 9 ]
+
+It works! Congratulations, you have built a new OCI container from scratch that uses bash scripting. Let's add some more configuration information.
+
+    buildah config --created-by "ipbabble"  $newcontainer
+    buildah config --author "wgh at redhat.com @ipbabble" --label fedora26-bashecho:latest $newcontainer
+ 
+We can inspect the container's metadata using the `inspect` command:
+ 
+    buildah inspect $newcontainer
+
+We should probably unmount and commit the image:
+
+     buildah unmount $newcontainer
+     buildah commit $newcontainer fedora-bashecho
+     buildah images
+     
+And you can see there is a new image called `fedora-bashecho:latest`. You can inspect the new image using:
+
+    buildah inspect --type=image fedora-bashecho
+
+## OCI images built using buildah are portable
+
+Let's test if this new OCI image is really portable to another OCI technology like docker. First you shouuld install docker and start it.
+
+    dnf -y install docker
+    systemctl start docker
+    
+The we need to push the image from the `containers/image` across to docker's storage area which is under `/var/lib/docker`. This is managed by the docker dameon. We can do this telling buildah to push to the docker protocol called docker-daemon.
+
+    buildah push fedora-bashecho docker-daemon:fedora-bashecho:latest
+
+Underneath `buildah` the `containers/storage` copies the image blob over to the docker storage. This can take a little while. And usually you won't need to do this. If you're using `buildah` you are probably not using docker. This is just for demo purposes. Letr's try it:
+
+    docker run fedora-bashecho 
+    This is a new container from ipbabble [ 0 ]
+    This is a new container from ipbabble [ 1 ]
+    This is a new container from ipbabble [ 2 ]
+    This is a new container from ipbabble [ 3 ]
+    This is a new container from ipbabble [ 4 ]
+    This is a new container from ipbabble [ 5 ]
+    This is a new container from ipbabble [ 6 ]
+    This is a new container from ipbabble [ 7 ]
+    This is a new container from ipbabble [ 8 ]
+    This is a new container from ipbabble [ 9 ]
+    
+OCI container images built with `buildah` are completely standard as expected. So now it might be time to run:
+
+    dnf -y remove docker
+
+## Using Dockerfiles with buildah
+
+What if you have been using docker for while and have some exisiting Dockerfiles. Not a problem. `buildah` can build images using a Dockerfile. The `build-using-dockerfile`, or `bud` for short, takes a Dockefile as input and produces an OCI image.
+
+Find one of your Dockerfiles or create a file called Dockerfile. Use the following example or some variation if you'd like:
+
+    # Base on the Fedora
+    FROM fedora:latest
+    MAINTAINER W Henry email ipbabble@gmail.com
+
+    # Update image and install hhtpd
```

Comment:
typo: httpd

Reply:
I noticed last night that my fingers kept doing this. I guess I didn't fix enough of them. 
ack done